src/misc/programs.cpp

```diff
@@ -566,7 +568,10 @@ bool set_ver(char *s), GFX_IsFullscreen(void);
 void CONFIG::Run(void) {
 	static const char* const params[] = {
 		"-r", "-wcp", "-wcd", "-wc", "-writeconf", "-wcpboot", "-wcdboot", "-wcboot", "-writeconfboot", "-bootconf", "-bc",
-		"-l", "-rmconf", "-h", "-help", "-?", "-axclear", "-axadd", "-axtype", "-get", "-set",
+		"-l", "-rmconf", "-h", "-help", "-?", "-axclear", "-axadd", "-axtype",
+		"-avistart","-avistop",
```

Comment:
So now DOSBox SVN has CONFIG -avistart and DOSBox-X has DX-CAPTURE :+1: Works for me.